docs/src/animation.md

```diff
@@ -61,6 +59,59 @@ A more complicated example:
 
 @example_database("Record Video")
 
+## Appending data to a plot
+
+If you're planning to append to a plot, like a `lines` or `scatter` plot (basically, anything that's point-based),
+you will want to pass an `Observable` Array of [`Point`](@ref)s to the plotting function, instead of passing `x`, `y`
+(and `z`) as separate Arrays.
+This will mean that you won't run into dimension mismatch issues (since Observables are synchronously updated).
+
+TODO add more tips here
+
+## Animating a plot "live"
```

Comment:
Should we remove this section entirely, since the `push!` syntax is no longer endorsed?